### PR TITLE
Add restriction on username start characters

### PR DESCRIPTION
**Pull request by RonanJackson**
Fixes https://github.com/GaloyMoney/galoy-mobile/issues/196

Worth noting someone might need a proper Spanish translation as this was just a quick Google Translate, thanks!

**Changes**
Now shows an error message when trying to create a username beginning with 'bc1', '1', or '3'.
Screenshot 2021-09-23 at 4 31 57 pm

### CODE REVIEW

File: app/i18n/es.json [hunk 1]
```diff
@@ -626,6 +626,7 @@
     "confirmSubtext": "Este nombre de usuario es permanente y no se podrá cambiar más adelante.",
     "confirmTitle": "¿Desea establecer %{input} como su nombre de usuario?",
     "letterAndNumber": "Solo se aceptan letras en minúscula, números y guiones bajos (_)",
+    "forbiddenStart": "No se puede empezar con bc1, 1, o 3 ",
```
Comment (dolcalmi) [inline app/i18n/es.json:629]: maybe is better to use "No debe empezar con bc1, 1 o 3 "

**Review by dolcalmi — CHANGES_REQUESTED** [review]

File: app/i18n/es.json (hunk 1 above)
Comment (jrbuhl93) [reply app/i18n/es.json:629]: Also list "forbiddenStart" before "letterAndNumber" to maintain alphabetical order.

**Comment by jrbuhl93** [comment]
Thanks for the pr!

File: app/i18n/es.json (hunk 1 above)
Comment (RonanJackson) [reply app/i18n/es.json]: Done!

**Pull request merged**